### PR TITLE
Reorganising folder structure

### PR DESCRIPTION
**Pull request by Ritika-gupta20**
1. Created folders: (a) test/Go1PercentFEAutomation-.>TECHHUB
 (b)  test/Go1PercentFEAutomation-.>TECHHUB
(c) page-objects->TECHHUB->AdminUser

2. Deleted the previous folders for Techhub(Admin User) FE and API
3. All the tests are running however a few bugs are discovered that have been cascaded to Lokeshwaran.
4. Github actions working at chromedriver version 118 only that has been updated in the package.json file.

### CODE REVIEW

File: .github/workflows/Go1Percent.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+        cache-dependency-path: './go1percent/package-lock.json'
+
+    - name: Clear node modules
+      run: rm -rf node_modules
```
Comment (Sparsh79) [inline .github/workflows/Go1Percent.yml:33]: I don't think this is required, as we are not going to commit node modules to the repo. I believe gitignore should take care of this.

File: .github/workflows/Go1Percent.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    - name: Clear package-lock.json
+      run: rm -f package-lock.json
+      
+    - name: Clear package-lock.json
+      run: npm cache clean --force
```
Comment (Sparsh79) [inline .github/workflows/Go1Percent.yml:39]: Duplicate steps, I think we can remove the second one.

File: .github/workflows/Go1Percent.yml (hunk 1 above)
Comment (Ritika-gupta20) [reply .github/workflows/Go1Percent.yml:33]: Okay will remove these

File: .github/workflows/Go1Percent.yml (hunk 2 above)
Comment (Ritika-gupta20) [reply .github/workflows/Go1Percent.yml:39]: Yes will do that

**Review by Sparsh79 — APPROVED** [review]

**Review by deepansh257 — APPROVED** [review]

**Pull request merged**